js/playerview.js

```diff
@@ -1,20 +1,22 @@
 // React view of a PlayerState
 import React from "react";
-
 import Card from "./cardview.js"
 
 let Player = React.createClass({
   render() {
-    let handCards = this.props.playerState.hand.map(function (card, i) {
+    let mana = this.props.playerState.mana;  
+    let maxMana = this.props.playerState.maxMana;  
+    let handCards = this.props.playerState.hand.map(function (cardInfo, i) {
       return (
-          <Card state={card} key={i} />
+          <Card cardInfo={cardInfo} player={this.props.playerState} key={i} />
       );
-    });
+    }.bind(this));
```

Comment:
I think in es6 you can do this with a => so you don't need to do bind(this) any more.